business/istio_config.go

```diff
@@ -328,7 +328,14 @@ func (in *IstioConfigService) GetIstioConfigList(criteria IstioConfigCriteria) (
 	go func(errChan chan error) {
 		defer wg.Done()
 		if criteria.IncludeAuthorizationPolicies {
-			if ap, apErr := in.k8s.GetIstioObjects(criteria.Namespace, kubernetes.AuthorizationPolicies, criteria.LabelSelector); apErr == nil {
+			var ap []kubernetes.IstioObject
+			var apErr error
+			if kialiCache != nil && kialiCache.CheckIstioResource(kubernetes.AuthorizationPolicies) && kialiCache.CheckNamespace(criteria.Namespace) {
```

Comment:
I'm seeing code like this repeated a lot:

```
if kialiCache != nil && kialiCache.CheckIstioResource(kubernetes.AuthorizationPolicies) && kialiCache.CheckNamespace(criteria.Namespace)
```

or similar but does not check Istio resource type:

```
if kialiCache != nil && kialiCache.CheckNamespace(namespace)
```

Should we have a utility func that performs this? e.g.

```
func IsCached(kc KialiCache, ns string, rt string) (bool) {
    ok := kc != nil && kc.CheckNamespace(ns)
    if ok && rt != nil {
        ok = kc.CheckIstioResource(rt)
    }
    return ok
}
```

Reply:
Then that repeated code and be replaced with things like:

`if IsCached(kialiCache, criteria.Namespace)`

or

`if IsCached(kialiCache, criteria.Namespace, kubernetes.AuthorizationPolicies)`

Reply:
Yeah, good suggestion